zsh/configs/keybindings.zsh

```diff
@@ -10,4 +10,5 @@ bindkey "^R" history-incremental-search-backward
 bindkey "^P" history-search-backward
 bindkey "^Y" accept-and-hold
 bindkey "^N" insert-last-word
+bindkey "^B" push-line-or-edit
```

Comment:
`^B` collides with an already existing mapping, what do you think of using another mapping? Like `^Q` that was mentioned on Slack.